includes/class-wc-tracker.php

```diff
@@ -86,6 +86,30 @@ private static function get_last_send_time() {
 		return apply_filters( 'woocommerce_tracker_last_send_time', get_option( 'woocommerce_tracker_last_send', false ) );
 	}
 
+	/**
+	 * Test whether this site is a staging site according to the Jetpack criteria.
+	 *
+	 * With Jetpack 8.1+, it uses \Automattic\Jetpack\Status::is_staging_site, with older JP,
+	 * it uses Jetpack::is_staging_site.
+	 *
+	 * @return bool
+	 */
+	private static function is_jetpack_staging_site() {
+		$is_jp_staging = false;
+		if ( class_exists( '\Automattic\Jetpack\Status' ) ) {
+			// Preferred way of checking with Jetpack 8.1+.
+			$jp_status = new \Automattic\Jetpack\Status();
+			if ( is_callable( array( $jp_status, 'is_staging_site' ) ) && $jp_status->is_staging_site() ) {
```

Comment:
There are versions of Jetpack with `\Automattic\Jetpack\Status` before `is_staging_site` was added to it and would still need to use the old fallback.

Reply:
Thanks, fixed in 3dcb3e0.